swagger.json

```diff
@@ -959,6 +959,182 @@
                 }
             }
         },
+        "/fleet/unassigned_driving_segments": {
```

Comment:
I spoke with Derrek about this endpoint. My thoughts are:
 * We should not use verbs in the URL Path -- we should instead use HTTP methods to represent verbs. 
 * Since unassigned-driving-segments is a first class citizen in our system (or at least we want it to be), this makes sense as a top level entity. 
   * However, we should also support indexing into a resource of this collection using the resource ID, such as `/fleet/unassigned-driving-segments/:id`. 
   * We should support `annotations` and `pendingDriverId` assignments via `PATCH /fleet/unassigned-driving-segments/:id`.